src/components/views/AgreementForm.js

```diff
@@ -128,6 +129,41 @@ class AgreementForm extends React.Component {
     );
   }
 
+  renderPaneFooter() {
+    const { initialValues = {} } = this.props;
+
+    const startButton = (
+      <Button
+        buttonStyle="default mega"
+        id="clickable-cancel"
+        marginBottom0
+        onClick={this.props.handlers.onClose}
+      >
+        <FormattedMessage id="stripes-components.cancel" />
+      </Button>
+    );
+
+    const endButton = (
+      <Button
+        buttonStyle="primary paneHeaderNewButton"
```

Comment:
Should this be `primary mega`? [Asking based on the ui-users example](https://github.com/folio-org/ui-users/blob/master/src/UserForm.js#L274).